spikeinterface/core/waveform_extractor.py

```diff
@@ -1299,6 +1299,22 @@ def extract_waveforms(recording, sorting, folder=None,
     we: WaveformExtractor
         The WaveformExtractor object
 
+    Examples
+    --------
+    >>> import spikeinterface as si
+    
+    >>> # Extract dense waveforms and save to disk
+    >>> we = si.extract_waveforms(recording, sorting, folder="waveforms")
+
+    >>> # Extract dense waveforms on all spikes
+    >>> we = si.extract_waveforms(recording, sorting, folder="waveforms-all", max_spikes_per_unit=None)
+
+    >>> # Extract dense waveforms in memory
+    >>> we = si.extract_waveforms(recording, sorting, folder=None, mode="memory")
+
+    >>> # Extract sparse waveforms (with radius-based sparsity of 50um) and save to disk
+    >>> we = si.extract_waveforms(recording, sorting, folder="waveforms-sparse", mode="folder",
+    >>>                           sparse=True, num_spikes_for_sparsity=100, method="radius", radius_um=50)
```

Comment:
exelent.
maybe one more example to show on job_kwargs is working

Reply:
done!